### PR TITLE
prevent multiple layouts from loading in analysis view, including mul…

### LINKED ISSUE #1187: Session drop down created multiple times / multiple event handlers
Need to make the initialization of the analysis page idempotent - so multiple widgets and event handlers are not created if it is initialized more than once.

### PR DESCRIPTION
**Pull request by brentpicasso**
…tiple flyin handles; prevent error when no trackmap geopoints are available; prevent premature dismissing of session flyin panel

resolves #1187 , #1112 

sorry for the whitespace changes, my editor auto-cleaned the files.

### CODE REVIEW

File: autosportlabs/racecapture/geo/geopoint.py [hunk 1]
```diff
@@ -2,17 +2,31 @@
 RADIUS_EARTH_KM = 6371;
 
 class GeoPoint:
+    '''
```
Comment (ryandoherty) [inline autosportlabs/racecapture/geo/geopoint.py]: Just an FYI that the PEP8 standard is double quotes, not single. https://www.python.org/dev/peps/pep-0257/

File: autosportlabs/racecapture/views/analysis/flyinpanel.py [hunk 2]
```diff
@@ -64,41 +65,41 @@ class FlyinPanelException(Exception):
...
+
     def __init__(self, **kwargs):
         Builder.load_string(FLYIN_PANEL_LAYOUT)
         super(FlyinPanel, self).__init__(**kwargs)
         self.hide_decay = Clock.create_trigger(lambda dt: self.hide(), self.SESSION_HIDE_DELAY)
         Window.bind(mouse_pos=self.on_mouse_pos)
         Window.bind(on_motion=self.on_motion)
         Clock.schedule_once(lambda dt: self.show())
-    
+        self.shown_at = None
```
Comment (ryandoherty) [inline autosportlabs/racecapture/views/analysis/flyinpanel.py]: Seems like this is a private var? If so probably best to prefix with `_`.

Comment (brentpicasso) [inline autosportlabs/racecapture/views/analysis/flyinpanel.py]: will do

File: autosportlabs/racecapture/geo/geopoint.py (hunk 1 above)
Comment (brentpicasso) [inline autosportlabs/racecapture/geo/geopoint.py]: fixed here and elsewhere.

**Pull request merged**